file_helper.py

```diff
@@ -69,7 +69,7 @@ def cache_tile(tile, source_name):
 
 
 def get_sample_data_directory():
-    return os.path.join(get_plugin_directory(), "sample_data")
```

Comment:
Did you even read the name of the method? It should return the path do the sample data directory. I understand why you want to change that, but then we'll do it right and not quick and dirty. I'll fix it now.